paperqa/types.py

```diff
@@ -52,6 +56,9 @@ class Doc(Embeddable):
             "flag to overwrite fields from metadata when upgrading to a DocDetails"
         ),
     )
+    overwrite_citation_from_metadata: bool = Field(
+        default=True, description=("flag to overwrite the citation from metadata")
```

Comment:
Few comments here:
- Can you remove the extra parens
- Can you clarify what you mean by "metadata" here? I would maybe say "manifest" not "metadata"
- Can you document why the default is `True`? "Defaults to True because ..."

Reply:
So I created this flag because we already had `overwrite_fields_from_metadata`.

`overwrite_fields_from_metadata` would overwrite `doc_id`, `dockey` and `citation` (and maybe another field I don't remember) but I wanted to separate `citation` from that.

In some way I copied the  `overwrite_fields_from_metadata` flag and its default settings. Do you want me to rename this flag to `overwrite_fields_from_manifest`?

@mskarlin wtyt?

Reply:
Oh I see. Let's consolidate them into one field:

```python
    overwrite_fields_from_metadata: Collection[str] = Field(
        default_factory=lambda: {"key": "docname", "doc_id": "dockey"},
        description=(
            "(fill in)"
        ),
    )
```

Then for RAG QA and any relevant tests, we'd do `overwrite_fields_from_metadata={"key": "docname", "doc_id": "dockey", "citation"}`

Reply:
@mskarlin suggested the same thing, and I believe is the best approach.
I think the problem with that is that you would like to specify that in a `.csv` right? So specifying a set can be complicated.